library/src/main/java/com/schibsted/spain/barista/internal/viewaction/SwipeActions.kt

```diff
@@ -43,6 +55,12 @@ object SwipeActions {
           NormalizedLocation(from.first, from.second),
           NormalizedLocation(to.first, to.second),
           Press.FINGER))
+
+  @JvmStatic
+  fun swipeViewPager2Forward(): ViewAction = ViewPager2SwipeAction(direction = FORWARD)
```

Comment:
just thinking...

Should we add viewpager2 as part of viewpager swipe? aas other methods on barista do for list and recyclers for example

Reply:
I agree. It's just that swiping a ViewPager2 is a bit more complicated than swiping the old ViewPager, since ViewPager2 supports both horizontal/vertical directions (plus we always have "right-to-left" layout to consider). 

So if I were to add ViewPager2 directly in `SwipeActions.swipe(from: Pair<..>, to: Pair<..>)` I would have to figure out the pairs even before getting to this call. And that would make it even worse than what I have done today I think 🤷‍♂️ 
Now it is still the case that the user only do `swipeViewPagerForward/Back()` regardless of which view pager is being used.

But maybe I misunderstood your comment or I haven't thought this through enough :)

Reply:
Good. Thanks for the explanation

Reply:
I'll just add this even if it might be obvious. But a "swipe forward" means different things in ViewPager2 depending on the horizontal/vertical orientation of it. That's what I was trying to say above :)